localization/beluga_demo_bearing_localization/package.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>beluga_demo_bearing_localization</name>
+  <version>2.0.0</version>
+
+  <description>Demo of fiducial based localization using Beluga and bearing estimates.</description>
+
+  <maintainer email="glpuga@ekumenlabs.com">Gerardo Puga</maintainer>
+  <maintainer email="ivanpauno@ekumenlabs.com">Ivan Paunovic</maintainer>
```

Comment:
Is Ivan aware?

Reply:
I don't know :sweat_smile: I didn't want to remove anyone.

Summoning @ivanpauno.

Reply:
I have no idea what this package is about, feel free to remove me :sweat_smile:

Reply:
I have no idea what this package is about, feel free to remove me :sweat_smile:

Reply:
Done in https://github.com/Ekumen-OS/beluga-demos/pull/8/commits/83fae496e0f3a96bdb42cfa7625a0e217d47897f.